playwright/src/components/create-vm/catalog-template-components.ts

```
@@ -0,0 +1,1850 @@
+          await this.locator('[data-test-id="disk-boot-source-container-source-password"]').fill(
+            password,
+          );
+        }
+        break;
+      case 'PVC':
+        await this.robustClick(this.locator('[data-test-id="pvc-clone"]'));
+        break;
+      case 'Upload':
+        const uploadMenuItem = this.locator('.pf-v6-c-menu__item-main', {
+          hasText: 'Upload (Upload a new file to a PVC)',
+        });
```

Biome (2.5.1)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)